.travis.yml

```diff
@@ -26,3 +26,4 @@ jobs:
         - cd build
         - cmake ../KRFAnalysisPass
         - cmake --build .
+        - cd ../tests && ./runTests.py
```

Comment:
Nice. It probably makes sense to add the relevant `black` and `clang-format` lines for the unit tests to the `lint` stage of the CI as well.